### DISCUSSION TITLE
How do new releases get into conda-forge?

### DISCUSSION THREAD

**Discussion by nevdelap** [dbt-labs/dbt-snowflake#594] [General]
I'm interested to know so that I know when to expect it to drop, or if that it's not dropped already is that an issue that needs raising somewhere?

I see that 1.4.2 was copied from cf-staging (https://anaconda.org/conda-forge/dbt-snowflake), so maybe 1.5.0 is working its way through staging, but I don't have access to see that.

Any insights? No tearing rush though. Thanks in advance, Nev.

**Comment by nevdelap** [comment]
I've raised this: https://github.com/dbt-labs/dbt-snowflake/issues/601

**Comment by dbeatty10** [comment]
Thanks for opening this discussion @nevdelap !

@maresb described the general procedure here:
https://github.com/dbt-labs/dbt-snowflake/issues/601#issuecomment-1541480100

**Comment by nevdelap** [comment]
Thank you!
